### PR TITLE
Make it possible to run more than one python test module

### PR DESCRIPTION
**Pull request by perlpunk**
Before all functions would be imported to the global namespace.

Issue: https://progress.opensuse.org/issues/132656

I had to rename test.py to pythontest.py because otherwise python was complaining:

    ImportError: Module 'test' is not installed.

### CODE REVIEW

File: autotest.pm [hunk 1]
```diff
@@ -114,14 +114,29 @@ sub loadtest ($script, %args) {
...
+            for my \$func (qw(@mapped_functions)) {
+                if (grep { \$_ eq \$func } \@{ \$info{functions} }) {
+                    py_bind_func("${name}::\$func", "$name", \$func);
+                }
+            }
+            EOM
         $is_python = 1;
     }
+
+
```
Comment (okurz) [inline autotest.pm:139]: Unrelated change

**Comment by perlpunk** [comment]
I added a second test module in our unit tests, and I also run the tests now to make sure the two modules do different things.

**Comment by perlpunk** [comment]
Also I now map all python functions to perl because I realized there are much more functions we call which might be overridden by the test.

**Comment by perlpunk** [comment]
Blocked by #2344

**Review by kalikiana — APPROVED** [review]

**Comment by kalikiana** [comment]
@mergifyio rebase

**Review by okurz — APPROVED** [review]
